src/components/icon.tsx

```diff
@@ -4,9 +4,9 @@ import colors from "../assets/colors"
 import "../assets/fonts"
 import icons, { IconName } from "../assets/icons"
 
-export type FontName = string
+type FontName = string
 
-export interface IconProps extends React.HTMLProps<JSX.Element> {
+interface IconProps extends React.HTMLProps<JSX.Element> {
```

Comment:
In some cases, we need to export a Props interface for a Component, to extend the component for instance. But we probably don't need to for IconProps.